screen-orientation/onchange-event-subframe.html

```diff
@@ -2,28 +2,44 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 
-<iframe id="testIframe" sandbox="allow-scripts allow-same-origin"
-  style="display:none" src="resources/iframe-listen-orientation-change.html">
+<iframe
+  id="testIframe"
+  sandbox="allow-scripts allow-same-origin"
+  style="display:none"
+  src="resources/iframe-listen-orientation-change.html"
+>
 </iframe>
 
 <script>
-promise_test(async t => {
-  let orientations = [
-    'portrait-primary',
-    'portrait-secondary',
-    'landscape-primary',
-    'landscape-secondary'
-  ];
-  if (screen.orientation.type.includes('portrait')) {
-    orientations = orientations.reverse();
-  }
-  const messageWatcher = new EventWatcher(t, window, "message");
+  promise_test(async t => {
+    t.add_cleanup(async () => {
+      try {
+      await document.exitFullscreen();
+      } catch (e) {}
+      screen.orientation.unlock();
+    });
+    await test_driver.bless("request fullscreen", () => {
+      return document.documentElement.requestFullscreen();
+    });
+    let orientations = [
```

Comment:
Apologies, I reformatted this file without realizing.... everything below changed is just prettier doing its thing.